dev-server/templates/ui-playground.mustache

```diff
@@ -4,7 +4,6 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>UI component playground</title>
-  <link rel="stylesheet" href="{{ resourceRoot }}/styles/sidebar.css">
```

Comment:
I want to avoid side effects from bits and bobs in the sidebar CSS affecting individual components. Adding stuff in modularly in `ui-playground`'s SCSS source for now.